.env

```diff
@@ -2,7 +2,8 @@ BUILD_VERSION=0.0.1
 CONTAINER_REGISTRY_SERVER=localhost:5000
 CONTAINER_REGISTRY_USERNAME=username
 CONTAINER_REGISTRY_PASSWORD=password
-REDIS_HOST_NAME=redisedge
-REDIS_EXPORTER_HOST_NAME=redisexporter
-PROMETHEUS_HOST_NAME=prometheus
-REDIS_TIME_SERIES_EDGE_HOST_NAME=redistimeseriesedge
\ No newline at end of file
+REDIS_UNIX_SOCKET_DIR=/tmp/docker
+REDIS_UNIX_SOCKET_VOL=redisUnixSocketVol
```

Comment:
The volume name is not special, it can be anything. However it must be the same for all containers where the bind is needed. The idea is to have a shared volume accessible to the Redis container and all other containers that need to access Redis.

The directory path can also be anything. However the difference is that this is a path inside the container so in theory it can be different for each mount (as long as volume name is the same). By using the `REDIS_UNIX_SOCKET_DIR` I've kept it the same just for simplicity.